src/main/java/org/fcrepo/spec/testsuite/crud/HttpPatch.java

```diff
@@ -122,12 +128,49 @@ public void serverManagedPropertiesModification(final String uri) {
                                         + "4.2.4.1), the server must fail the request by responding with a 4xx range"
                                         + " status code (e.g. 409 Conflict).",
                                         "https://fcrepo.github.io/fcrepo-specification/#http-patch", ps);
+
+        //This test assumes that creating an empty container will result in RDF that contains at least one
+        // server-managed triple.
         final Response resource = createBasicContainer(uri, info);
-        final String locationHeader = getLocation(resource);
+        final String resourceUri = getLocation(resource);
+
+        //parse the resource's body
+        final Model model = createDefaultModel();
+        model.read(doGet(resourceUri, new Header("Accept", "text/turtle")).getBody().asInputStream(), "",
+                   "text/turtle");
+        final Dataset dataset = new DatasetImpl(model);
+        final DatasetGraph graph = dataset.asDatasetGraph();
+        final Iterator<Quad> it = graph.find();
+        boolean atLeastOneUpdateFailed = false;
+
+        while (it.hasNext()) {
+            //for each triple
+            final Triple oldValue = it.next().asTriple();
+            final String newObject = "newValue:" + System.currentTimeMillis();
+            //attempt to update the resource with that triple
+            if (patchReturned4xx(resourceUri, oldValue.getPredicate().getURI(),
```

Comment:
If another implementation requires certain fields to have certain types of object values (lastModified must be a date, or ldp:contains must be a URI) this would fail due to converting everything to a string literal instead of the server managed triple restriction. On the other hand, the language of this section just says "request is received that attempts to modify statements to a resource that a server disallows", so maybe this falls into the same boat.

Reply:
Yes, Trellis has restrictions on certain values (e.g. neither `ldp:membershipResource` nor `rdf:type` can refer to a `Literal`). I think you'd be best served by looking at the restrictions that LDP already defines and testing those.